libguard/guard_file.cpp

```diff
@@ -57,14 +52,14 @@ void GuardFile::read(const uint64_t pos, void* dst, const uint64_t len)
                   "Unable to move to the position in the guard file at"
                   " position= 0x%016llx",
                   pos);
-        throw std::runtime_error("Failed to move to the position in the file");
+        throw InvalidEntityPath("Failed to move to the position in the file");
```

Comment:
Why throwing `InvalidEntityPath` exception when failed to seek the guard file?

Reply:
Added GuardSeekFailed